pallets/settlement/src/benchmarking.rs

```diff
@@ -789,7 +789,7 @@ benchmarks! {
         // Add instruction
         Module::<T>::base_add_instruction(did, venue_id, SettlementType::SettleOnAffirmation, None, None, legs.clone())?;
         let instruction_id = 1;
-        let ticker = Ticker::try_from(generate_ticker(1u64).as_slice()).unwrap();
+        let ticker = Ticker::try_from(Ticker::generate(1u64).as_slice()).unwrap();
```

Comment:
Perhaps in a different PR but we can directly use `Ticker::generate_into` in all of these calls.